src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -450,7 +450,8 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
                 analyzedNullabilityMapOpt: null,
                 updatedMethodSymbolMapOpt: null,
                 snapshotBuilderOpt: null,
-                returnTypesOpt: null);
+                returnTypesOpt: null,
+                out _);
```

Comment:
>out [](start = 16, length = 3)

nit: consider naming this argument

Reply:
No longer relevant.

---
In reply to: [305056178](https://github.com/dotnet/roslyn/pull/37322#discussion_r305056178) [](ancestors = 305056178)